File: src/c#/10Print.cs
using System;


namespace AkashGutha  
{
    class TenPrint  
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("hello world!!");

            double iterations = 1 << 16;
            Random r = new Random();

            while (iterations-- > 0)
            {
                int next = r.Next(0,11);
                if(next <= 5)
                {
                    Console.Write("\\");
                }
                else
                {
                    Console.Write("/");
                }
            }

        }
    }
}

File: src/python/10Print.py
# 10Print in python
import random

Iterations = 1 << 16
Random = 0

while Iterations > 0:
    Random = random.randrange(0, 10, 1)

    # random generation
    if(Random >= 5):
        print("/", end="")
    else:
        print("\\", end="")

    Iterations = Iterations - 1
